website/sidebars.js

```diff
@@ -172,7 +172,21 @@ module.exports = {
                                 'api/internal/health',
                             ],
                         },
-                        'api/open_api',
+                        // NOTE: activate this when we're ready to integrate generated OpenAPI docs
+                        //
+                        // {
+                        //     label: 'OpenAPI docs',
+                        //     collapsed: true,
+                        //     type: 'category',
+                        //     link: {
+                        //         title: 'Unleash Server APIs',
+                        //         type: 'generated-index',
+                        //         description:
+                        //             'Generated API docs based on the Unleash OpenAPI schema. For the time being, some additional info can also be found in the older API docs.',
+                        //         slug: '/reference/apis/unleash',
+                        //     },
+                        //     items: require('./docs/reference/apis/unleash/sidebar.js'),
+                        // },
```

Comment:
This adds OpenAPI docs to the sidebar.

Reply:
Nice, is this everything we have to comment out? The previous docs will still be there?